network/p2p/gossip/gossip_test.go

```diff
@@ -13,8 +13,6 @@ import (
 
 	"github.com/stretchr/testify/require"
 
-	"go.uber.org/mock/gomock"
```

Comment:
These changes are purely aesthetic because I started finding the mocking libraries frustrating to use